modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -1702,40 +1702,37 @@ sub _munge_meta {
     $self->full_tree->{'MULTI'}{'SPECIES_ALIASES'}{$species} = $species;
 
     ## Used mainly in <head> links
-    ($self->tree->{'SPECIES_BIO_SHORT'} = $self->tree->{'SPECIES_URL'}) =~ s/^([A-Z])[a-z]+_([a-z]+)$/$1.$2/;
+    ($self->tree($production_name)->{'SPECIES_BIO_SHORT'} = $self->tree->{'SPECIES_URL'}) =~ s/^([A-Z])[a-z]+_([a-z]+)$/$1.$2/;
     
-    if ($self->tree->{'ENSEMBL_SPECIES'}) {
-      push @{$self->tree->{'DB_SPECIES'}}, $species;
-    } else {
-      $self->tree->{'DB_SPECIES'} = [ $species ];
-    }
+    push @{$self->tree->{'DB_SPECIES'}}, $species;
```

Comment:
Just checking if we still need the if statement from the left.

Reply:
No - the arrayref will autovivify. What we do need is to remove this, as it duplicates line 1708.

Reply:
Actually we do possibly need to add DB_SPECIES twice, both to the species tree and to the collection tree. It's harmless to non-collection databases and may be needed by code elsewhere.